src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/register_app_interface_request_test.cc

```diff
@@ -461,12 +461,6 @@ TEST_F(RegisterAppInterfaceRequestTest, Run_MinimalData_SUCCESS) {
       .WillByDefault(
           Return(application_manager::HmiInterfaces::HMI_INTERFACE_Buttons));
 
```

Comment:
```suggestion

EXPECT_CALL(mock_hmi_capabilities_, is_ui_cooperating())
      .WillRepeatedly(Return(true));

  MessageSharedPtr button_caps_ptr(CreateMessage(smart_objects::SmartType_Map));
  (*button_caps_ptr)[0][am::hmi_response::button_name] =
      mobile_apis::ButtonName::CUSTOM_BUTTON;

  EXPECT_CALL(mock_hmi_capabilities_, button_capabilities())
      .WillRepeatedly(Return(button_caps_ptr));

  EXPECT_CALL(
      mock_rpc_service_,
      ManageHMICommand(
          HMIResultCodeIs(hmi_apis::FunctionID::Buttons_SubscribeButton), _))
      .WillOnce(Return(true));

```

indentation in suggestion is wrong, but I think it would be valuable to have at least one unit test for the Request being sent.

Reply:
@iCollin fixed in https://github.com/smartdevicelink/sdl_core/pull/3744/commits/a9e960dfcedf27878d57d64ce9c7108b5fc73781